polymer.externs.js

```diff
@@ -666,10 +739,84 @@ Polymer.Templatizer = {
   /**
    * @param {?Element} template
    */
-  templatize: function(template) {}
+  templatize: function(template) {},
+
+  /**
+   * Returns the template "model" associated with a given element, which
+   * serves as the binding scope for the template instance the element is
+   * contained in. A template model is an instance of `Polymer.Base`, and
+   * should be used to manipulate data associated with this template instance.
+   *
+   * Example:
+   *
+   *   var model = modelForElement(el);
+   *   if (model.index < 10) {
+   *     model.set('item.checked', true);
+   *   }
+   *
+   * @param {!HTMLElement} el Element for which to return a template model.
+   * @return {Object<PolymerElement>} Model representing the binding scope for
```

Comment:
Not sure this type is right here (I just copied it from the docs). Do you know what type this actually is? We should also add either ! or ? to the front of both Object and PolymerElement.